### DISCUSSION TITLE
Title is not being shown on android

### DISCUSSION THREAD

**Discussion by elie-chaaban** [ctwillie/expo-server-sdk-php#17] [Q&A] [answered]
Title is not being shown on android

**Comment by cedrictwillie** [comment, marked as answer]
@elie-chaaban Can you show an example of how you are setting the title property on the `ExpoMessage` class?

**Reply by elie-chaaban** [reply]
It worked man, I was missing something! Thanks

**Reply by cedrictwillie** [reply]
Ok, awesome!
